app/(storyblok)/[...slug]/page.tsx

```diff
@@ -121,18 +118,18 @@ export async function generateMetadata({ params }: { params: ParamsType }): Prom
   try {
     const { data } = await getStoryData(params);
     if (!data.story || !data.story.content) {
-      throw new Error(`No story data found for ${params.slug.join('/')}`);
+      notFound();
```

Comment:
This throws an error but a special 404 error that gets picked up at the error boundary and returns the `not-found.tsx` file.